src/app/views.py

```diff
@@ -218,8 +223,14 @@ def parseXmlString(xmlString):
         if(len(tree.findall('{http://www.spdx.org/license}license/{http://www.spdx.org/license}text')) > 0):
             textElem = tree.findall('{http://www.spdx.org/license}license/{http://www.spdx.org/license}text')[0]
             ET.register_namespace('', "http://www.spdx.org/license")
-            textStr = ET.tostring(textElem)
-            data['text'] = textStr
+            textStr = ET.tostring(textElem).strip()
+            if(len(textStr) >= 49 and textStr[:42] == '<text xmlns="http://www.spdx.org/license">' and textStr[-7:] == '</text>'):
+                textStr = textStr[42:]
+                textStr = textStr[:-7].strip().replace('&lt;', '<').replace('&gt;', '>').strip()
+            if(len(textStr) >= 12 and textStr[0:9] == '<![CDATA[' and textStr[-3:] == ']]>'):
+                textStr = textStr[9:]
+                textStr = textStr[:-3]
+            data['text'] = textStr.strip()
```

Comment:
I'm not understanding the reason for adding the CDATA and removing it here.

Adding the `<br />` tags should be valid HTML.

I also tested this out and the CDATA is present in the downloaded XML.

Reply:
I would treat the text entered by the user just as plain text.  We should escape the text for HTML (e.g. `<` -> `&lt;`) or any other characters which may not be parseable by the XML.